tools/windows-node-installer/cmd/aws.go

```diff
@@ -81,11 +84,12 @@ func createCmd() *cobra.Command {
 		RunE: func(_ *cobra.Command, args []string) error {
 			cloud, err := cloudprovider.CloudProviderFactory(rootInfo.kubeconfigPath, awsInfo.credentialPath,
 				awsInfo.credentialAccountID, rootInfo.resourceTrackerDir,
-				awsInfo.imageID, awsInfo.instanceType, awsInfo.sshKey)
+				awsInfo.imageID, awsInfo.instanceType, awsInfo.sshKey, awsInfo.privateKeyPath)
 			if err != nil {
 				return fmt.Errorf("error creating aws client, %v", err)
 			}
-			err = cloud.CreateWindowsVM()
+			// TODO: Use the credentials object and print password, user name etc here.
```

Comment:
We should abstract this out as in the calls to `cloud.CreateWindowsVM()` and `DestroyWindowsVMs()` should happen at a higher level.